### PR TITLE
Added result numbers to the FindBox widget.

### PR DESCRIPTION
**Pull request by Kirottu**
Added the amount of results found for the query into the `FindBox` widget in this fashion:
image

Does not relate to any github issue I can find, but was a thing someone was looking for.

### CODE REVIEW

File: lapce-ui/src/find.rs [hunk 1]
```diff
@@ -205,6 +234,45 @@ impl Widget<LapceTabData> for FindBox {
...
+        }
+
+        let text_layout = ctx
+            .text()
+            .new_text_layout(if buffer.doc.find.borrow().occurrences().len() > 0 {
+                format!(
+                    "{}/{}",
+                    match index {
+                        Some(index) => format!("{}", index + 1),
+                        None => "?".to_string(),
```
Comment (dzhou121) [inline lapce-ui/src/find.rs:254]: It looks a bit odd like ```?/215```, maybe we just show ``` 215 found ``` instead?

Comment (Kirottu) [reply lapce-ui/src/find.rs:254]: Hmmm yeah agreed, that would make more sense.

Comment (Kirottu) [reply lapce-ui/src/find.rs]: Now it shows `215 results` instead of `?/215`, so I presume this is resolved.

**Comment by dzhou121** [comment]
Thanks!

**Pull request merged**